### ISSUE TITLE
Exception when using Python SDK for Kratos verification

### ISSUE THREAD

**Issue by AntiSC2** [ory/sdk#11]
**Describe the bug**

This is very similar to ory/sdk#10. An exception is thrown because Kratos sets the `verified_at`-field to `null` which causes an exception to be thrown in the Python SDK.

**Reproducing the bug**

This happens when using the Python SDK and calling the function `api_instance.list_identities()`. If a user is unverified then the `verified_at`-field is `null` and the exception is thrown.

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/tornado/web.py", line 1703, in _execute
    result = await result
  File "/app/app/web/handlers/admin/members.py", line 24, in get
    members = api_instance.list_identities()
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api/admin_api.py", line 925, in list_identities
    return self.list_identities_with_http_info(**kwargs)  # noqa: E501
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api/admin_api.py", line 987, in list_identities_with_http_info
     return self.api_client.call_api(
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 348, in call_api
    return self.__call_api(resource_path, method,
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 192, in __call_api
    return_data = self.deserialize(response_data, response_type)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 264, in deserialize
    return self.__deserialize(data, response_type)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 280, in __deserialize
    return [self.__deserialize(sub_data, sub_kls)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 280, in <listcomp>
    return [self.__deserialize(sub_data, sub_kls)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 303, in __deserialize
    return self.__deserialize_model(data, klass)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 639, in __deserialize_model
    kwargs[attr] = self.__deserialize(value, attr_type)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 280, in __deserialize
    return [self.__deserialize(sub_data, sub_kls)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 280, in <listcomp>
    return [self.__deserialize(sub_data, sub_kls)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 303, in __deserialize
    return self.__deserialize_model(data, klass)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/api_client.py", line 641, in __deserialize_model
    instance = klass(**kwargs)
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/models/verifiable_address.py", line 71, in __init__
    self.verified_at = verified_at
  File "/usr/local/lib/python3.8/site-packages/ory_kratos_client/models/verifiable_address.py", line 185, in verified_at
    raise ValueError("Invalid value for `verified_at`, must not be `None`")  # noqa: E501
ValueError: Invalid value for `verified_at`, must not be `None`
```

**Expected behavior**

The SDK should be able to handle the value being null.

**Comment by aeneasr** [comment]
Thank you for helping cleaning this up! If you want, to fix this, you need to remove the `required: true` here: https://github.com/ory/kratos/blob/master/identity/address.go#L44-L45

Basically this error happens when `required` is incorrectly set to true.

**Comment by AntiSC2** [comment]
I have opened up a pull request with a fix, ory/kratos#353
